lib/unittest.bzl

```diff
@@ -177,7 +179,7 @@ def _make_analysis_test(impl, expect_failure = False, config_settings = {}):
       A rule definition that should be stored in a global whose name ends in
       `_test`.
     """
-    attrs = {}
+    attrs = dict(attrs) if attrs else {}
```

Comment:
Could you just make the default value of attrs be {} instead of None, and then just say attr = dict(attrs) here for mutability?
(Wouldn't be a big deal except Stardoc thus automatically detects that attrs is expected to be a dict :))

Reply:
do you want me to make this change in unittest.make as well?

Reply:
ooh, I hadn't noticed we also did that there. yes please!

Reply:
done!